Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1635,7 +1635,7 @@ inline void BuilderCustom::applyInitialFontVariantAlternates(BuilderState& build
 
 inline void BuilderCustom::applyValueFontVariantAlternates(BuilderState& builderState, CSSValue& value)
 {
-    auto setAlternates = [&builderState](FontVariantAlternates alternates) {
+    auto setAlternates = [&builderState](FontVariantAlternates&& alternates) {
```

Comment:
Given how it is used, I think I would have used `const FontVariantAlternates&` here..

Reply:
Will do!